### PR TITLE
Fix three NPE.

### PR DESCRIPTION
**Pull request by PirolA**
Added null checks.

We want to come back to substance (from insubstantial) and these have been issues, which have been fixed.

### CODE REVIEW

**Comment by kirill-grouchnikov** [comment]
It's impossible to see what changed in the web revision. The entirety of two files is marked as changed.

**Comment by PirolA** [comment]
Our "corporate workspace configuration" in eclipse changed all LF to CRLF before comitting. I'll fix that.

**Comment by PirolA** [comment]
It's still messy. I'll try again...

File: src/org/pushingpixels/substance/internal/ui/SubstanceTableUI.java [hunk 1]
```diff
@@ -993,8 +994,10 @@ protected void paintCell(Graphics g, Rectangle cellRect, Rectangle highlightCell
             rendererComponent = this.table.prepareRenderer(renderer, row, column);
             boolean isSubstanceRendererComponent = isSubstanceDefaultRenderer(rendererComponent);
             if (isSubstanceRenderer && !isSubstanceRendererComponent) {
-                throw new IllegalArgumentException(
+            	if (!Boolean.getBoolean("substance.looseTableCellRenderers")) {
```
Comment (kirill-grouchnikov) [inline src/org/pushingpixels/substance/internal/ui/SubstanceTableUI.java]: This is not going to make it in.

Comment (PirolA) [reply src/org/pushingpixels/substance/internal/ui/SubstanceTableUI.java]: The following issue let me create this pull request
https://github.com/Insubstantial/insubstantial/issues/63

**Comment by PirolA** [comment]
All whitespaces should be fixed ;)

File: src/org/pushingpixels/substance/internal/ui/SubstanceTableUI.java (hunk 1 above)
Comment (kirill-grouchnikov) [reply src/org/pushingpixels/substance/internal/ui/SubstanceTableUI.java]: I'm not going to comment on decisions made in the project fork.

If this is an issue in your app, it needs to be addressed properly - in the context of the planned enhancement for https://github.com/kirill-grouchnikov/substance/issues/66

There will be no one-off properties like this in the codebase.

**Comment by PirolA** [comment]
I rebased again to remove meaningless commits and removed that boolean flag.

**Comment by kirill-grouchnikov** [comment]
This will be added as part of 8.0.01 tonight or tomorrow.

**Pull request merged**